x/kavamint/types/genesis.go

```diff
@@ -1,10 +1,14 @@
 package types
 
 import (
-	fmt "fmt"
 	time "time"
 )
 
+var (
+	// DefaultPreviousBlockTime represents a time that is unset -- no previous block has occured
+	DefaultPreviousBlockTime = time.Time{}
```

Comment:
change of default to assume no previous block and no accumulation on the first height after initgenesis

@pirtleshell we can set a previous block time explicity in the upgrade handler if we want